examples/container-service-extension/3.7-cse-step3.tf

```diff
@@ -0,0 +1,20 @@
+# --------------------------------------------------------------------------------------------------------
+# WARNING: This is done after running `cse install` command.
```

Comment:
What is the reason it's not together inside of step2? Actually, would be good to add that explanation to the comment here, because right now the warning is identical to step2 and it's not clear why it's in a separate .tf file.

Reply:
You're right, I've moved this to step2 and changed the comments accordingly. Not sure why I separated them 🤔

Reply:
Nice, we fit in the three steps now :)